os/cluster-architectures.md

```diff
@@ -60,22 +49,43 @@ coreos:
+
+Once the local VM is running, tell your Docker binary to use the remote port by exporting an environment variable and start running Docker commands:
+
+```
+$ export DOCKER_HOST=tcp://localhost:2375
+$ docker ps
+```
+
+This avoids potentially breaking differences between your local development environment and where the container will *actually* run in production.
+
+### Related local installation tools
+
+There are a myriad of alternative solutions for testing CoreOS locally:
+
+- [coreos-vagrant][coreos-vagrant] is not an officially fully supported platform, however may be a good resource to check out if you are already comfortable with Vagrant and are unable to use one of ignition's officially supported platforms.
+- [coreos-kubernetes][coreos-kubernetes] provides resources to run a local Kubernetes cluster with one or more nodes provisioned by Vagrant.
+- [Minikube][minikube] is used for local Kubernetes development. This does not use CoreOS but is very fast to setup and is the easiest way to test-drive use Kubernetes.
+
 ## Small cluster
 
 <img class="img-center" src="img/small.png" alt="Small Container Linux Cluster Diagram"/>
 <div class="caption">Small Container Linux cluster running etcd on all machines</div>
 
-| Cost | Great For | Set Up Time | Production |
-|------|-----------|-------------|------------|
-| Low | Small clusters, trying out Container Linux | Minutes | Yes |
+| Cost | Great For                                  | Set Up Time | Production |
+|------|--------------------------------------------|-------------|------------|
+| Low  | Small clusters, trying out Container Linux | Minutes     | Yes        |
 
-For small clusters between 3-9 machines, running etcd on all of the machines allows for high availability without paying for extra machines that just run etcd.
+For small clusters, between 3-9 machines, running etcd on all of the machines allows for high availability without paying for extra machines that just run etcd.
 
-Getting started is easy &mdash; a single cloud-config can be used to start all machines on a cloud-provider.
+Getting started is easy &mdash; a single Container Linux config can be used to provision all machines on your cloud-provider.
 
 ### Configuring the machines
 
-Following the guide for each of the [supported platforms](https://coreos.com/docs#running-coreos) will be the easiest way to get started with this architecture. Boot the desired number of machines with the same cloud-config and discovery token. The cloud-config specifies that etcd and fleet will be started on each machine.
+Following the guide for each of the [supported platforms](https://coreos.com/docs#running-coreos) will be the easiest way to get started with this architecture. Boot the desired number of machines with the same Container Linux config and discovery token. The Container Linux config specifies that etcd and other services will be started on each machine.
```

Comment:
For more information on getting started with this architecture, see the CoreOS documentation on ...

call out the specific reference, rather than just 'supported platforms', please. Given the location of the reference in this doc, and the info in the doc you're pointing to, I really don't know what you want me to look at.